keg_elements/templates/keg_elements/forms/horizontal_b4.html

```diff
@@ -127,9 +127,10 @@
  #}
 {% macro checkbox_field(field) -%}
     {% call div_form_group(field, **kwargs) %}
-        <div class="unlabeled-group checkbox">
-            <label>
-                {{ field(type='checkbox', **kwargs) }} {{ field.label }}
+        <div class="unlabeled-group checkbox form-check">
```

Comment:
What is this `unlabeled-group`, I can't seem to find it in the documentation or in our styles.